src/components/Comments/content.js

```diff
@@ -83,10 +83,24 @@ const ContentComments = () => {
                     <div className="commentheader"><h3 className="headertext">Rate and Review</h3></div>
                     <Row>
                         <Form className="formcontentname" onSubmit={handleFormSubmit} >
+                            <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
+                                <Form.Label>Email address: </Form.Label>
+                                <Form.Control type="email" placeholder="name@example.com" />
+                            </Form.Group>
+                            <Form.Group>
+                                <Form.Label>Rating: </Form.Label>
+                            <ReactStars className="ratingStyles"
+                                count={5}
+                                onChange={ratingChanged}
+                                size={24}
+                                edit={true}
+                                color2={'#ffd700'} />
+                            </Form.Group>
```

Comment:
Oooooooo ReactStars looks fire and handy!! THANK YOU SO MUCH FOR THIS 🥇 sheeeeeeeeeeeesh